scripts/rebuild.sh

```diff
@@ -5,7 +5,8 @@ set -e
 cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")"
```

Comment:
We can get the GITHUB_TOKEN from the pipeline itself:
```yaml
- bash: rebuild.sh
  env:
    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```


https://dev.to/github/the-githubtoken-in-github-actions-how-it-works-change-permissions-customizations-3cgp
https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/